models/intermediate/int_zendesk__organization_aggregates.sql

```diff
@@ -11,13 +11,15 @@ with organizations as (
 ), tag_aggregates as (
     select
         organizations.organization_id,
+        organizations.source_relation,
```

Comment:
For some reason I am running into the following error when running on the `zendesk_yashwanth` schema. Do you have any ideas why this may be occurring? 

![image](https://github.com/fivetran/dbt_zendesk/assets/74217849/108109b6-0a0a-44e3-8b74-9a8be2c980b0)

Reply:
also not getting this error... could you rerun @fivetran-joemarkiewicz ?